generative/networks/blocks/encoder_modules.py

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from collections.abc import Sequence
+from functools import partial
+
+import torch
+import torch.nn as nn
+from monai.networks.blocks import Convolution
+
+__all__ = ["SpatialRescaler"]
+
+
+class SpatialRescaler(nn.Module):
+    """
+    SpatialRescaler based on https://github.com/CompVis/latent-diffusion/blob/main/ldm/modules/encoders/modules.py
+
+    Args:
+        spatial_dims: number of spatial dimensions.
+        n_stages: number of interpolation stages.
+        size: output spatial size (int or Tuple[int] or Tuple[int, int] or Tuple[int, int, int]).
+        method: algorithm used for sampling.
+        multiplier: multiplier for spatial size. If scale_factor is a tuple,
```

Comment:
this part about `scale_factor` doesn't make sense - there is no variable with that name, and multiplier can't be a tuple. maybe just delete the part about `scale_factor`?

Reply:
Thanks for pointing out the docstring problem. I have fixed it. This remembers me that the `multiplier` can be a `Sequence[float]`. I already added this type and a new test case for this situation in the latest commit. https://github.com/Project-MONAI/GenerativeModels/blob/01ff8dc52582e16714edb475b990c80b82035d25/tests/test_encoder_modules.py#L77